stanford/roles/cluster/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+CLUSTER_AMI: 'ami-09d2fb69'
+CLUSTER_DEPLOYMENT: sandbox
+CLUSTER_ELB_HEALTH_CHECK:
+  ping_path: '/'
+  ping_protocol: http
+  ping_port: 80
+  response_timeout: 5  # seconds
+  interval: 30  # seconds
+  unhealthy_threshold: 2  # attempts
+  healthy_threshold: 10  # attempts
+CLUSTER_ELB_LISTENERS:
+  - protocol: http
```

Comment:
not `https`?

Reply:
This is overridden elsewhere; it's just a placeholder.